aspnetcore/host-and-deploy/iis/troubleshoot.md

```diff
@@ -34,12 +44,38 @@ Learn about the debugging support built into Visual Studio Code.
 **502.5 Process Failure**  
 The worker process fails. The app doesn't start.
 
-The ASP.NET Core Module attempts to start the worker process but it fails to start. The cause of a process startup failure can usually be determined from entries in the [Application Event Log](#application-event-log) and the [ASP.NET Core Module stdout log](#aspnet-core-module-stdout-log).
+The ASP.NET Core Module attempts to start the backend dotnet process but it fails to start. The cause of a process startup failure can usually be determined from entries in the [Application Event Log](#application-event-log) and the [ASP.NET Core Module stdout log](#aspnet-core-module-stdout-log). 
+
+A common failure condition is the app is misconfigured due to targeting a version of the ASP.NET Core shared framework that isn't present. Check which versions of the ASP.NET Core shared framework are installed on the target machine.
 
 The *502.5 Process Failure* error page is returned when a hosting or app misconfiguration causes the worker process to fail:
 
 ![Browser window showing the 502.5 Process Failure page](troubleshoot/_static/process-failure-page.png)
 
+::: moniker range=">= aspnetcore-2.2"
+
+**500.30 In-Process Startup Failure**
+
+The worker process fails. The app doesn't start.
+
+The ASP.NET Core Module attempts to start the .NET Core CLR in-process but it fails to start. The cause of a process startup failure can usually be determined from entries in the [Application Event Log](#application-event-log) and the [ASP.NET Core Module stdout log](#aspnet-core-module-stdout-log). 
+
+A common failure condition is the app is misconfigured due to targeting a version of the ASP.NET Core shared framework that isn't present. Check which versions of the ASP.NET Core shared framework are installed on the target machine.
+
+**500.0 In-Process Handler Load Failure**
+
+The worker process fails. The app doesn't start.
+
+The ASP.NET Core Module fails to find the .NET Core CLR and find the in-process request handler (aspnetcorev2_inprocess.dll). Check that the app targets the <xref:Microsoft.AspNetCore.Server.IIS> nuget package or Microsoft.AspNetCore.App and check which versions of the ASP.NET Core shared framework are installed on the target machine.
```

Comment:
* Emphasis on the DLL ... \*aspnetcorev2_inprocess.dll\*
* You'll need to link directly to the package ... \[Microsoft.AspNetCore.Server.IIS](https://www.nuget.org/packages/Microsoft.AspNetCore.Server.IIS)
* We've standardized on "NuGet" capitalization
* Link the `App` package this way ... \[Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app)
* Due to the length, the last line should broken into a list of two items

Here's a cut-'n-paste ...

> The ASP.NET Core Module fails to find the .NET Core CLR and find the in-process request handler (\*aspnetcorev2_inprocess.dll\*). Check that:
>
> \* The app targets either the \[Microsoft.AspNetCore.Server.IIS](https://www.nuget.org/packages/Microsoft.AspNetCore.Server.IIS) NuGet package or the \[Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app).
> \* The version of the ASP.NET Core shared framework that the app targets is installed on the target machine.